dvc/ui/__init__.py

```diff
@@ -1,4 +1,7 @@
+import webbrowser
 from contextlib import contextmanager
+from pathlib import Path
+from platform import uname
```

Comment:
The `dvc.ui` module is meant to be imported globally as it's a singleton, so a great care has been taken to have almost no imports as possible. Please import this inside ' open_browser`.